interview-questions.md

```diff
@@ -7,60 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
-  Researched answer:
+  Researched answer: Git stands for Global Information Tracker and it is an open-source version control system. It is a software used by developers to track changes in files. It allows multiple developers to collaboratively develop and work on the same base code. It allows the entire history of the codebase to be accessible and available on every developer's computer. Github is an online git repository that stores code for developers to manage, track and control changes to the code.
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: Boolean is a JavaScript data type that evaluates data and returns a true or false value. The Boolean data type are primarily used for comparisons. For example; is (100>10)--> true or is (10>100)--> false.
 
-  Researched answer:
+  Researched answer:Boolean is a JavaScript data type used for comparisons that return a true or false value; they are not strings. There are three type of operators that will return a Boolean value: equality operators, relational operators, and logical operators. Equality operators compares two values and determines if they equate to the same. There are two types of equality operators. Loose equality operators ("==") will convert the two values to the same data type before comparing them. For example, when comparing a number to a string such as this (5 == "5"), the loose equality operator will convert the number and string to the same data type and will result to a true Boolean value. The strict equality operator is used to only compare data types that are the same by using the ("==="). So an evaluation that compares (5==="5") with the strict equality operator will return a a Boolean value of false because the two data types being compared are a string and a number. Relational operators look to see if one value is greater than (>), less than (<), greater than or equal to (>=), or less than or equal to (<=) another value. For example, is (10>=5)--> true. Logical operators use &&, ||, ! to compare values. The And operator returns a true value when it evaluates that both sides of the equation are true. For example is (10>=5 && 5>=0)--> true. If both sides of the equation are not equal then the evaluation will return a false value. The Or operator will only look to see if one side of the equation is at least true. Is (10>=5 || 0>=5)--> true; is (5>=10 || 0>=5)--> false. The bang operator will return the logical opposite. So if (!true)--> false.
```

Comment:
Excellent answer, Easily the most comprehensive one I have come across. Please consider putting this in a blog post about your coding journey.